### PR TITLE
#4969 [Manager] Navigator item's active/selected state background col…

### PR DESCRIPTION
**Pull request by konekoya**
…or is not taking full width

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4969)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/common/Dialog/Dialog.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ Dialog.propTypes = {
 
 Dialog.defaultProps = {
   confirmDisabled: false,
-  confirmTooltip: null,
+  confirmTooltip: '',
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Dialog/Dialog.js:143]: @eechih This is causing component using this without `confirmTooltip` prop to complain about the prop is required. Changing this to an empty string fixes the issue. This is the behavior form the docs: "Tooltip title. Zero-length titles string are never displayed." :)

https://material-ui.com/api/tooltip/

**Comment by konekoya** [comment]
retry manager-e2e

File: ohara-manager/client/src/components/common/Dialog/Dialog.js (hunk 1 above)
Comment (eechih) [reply ohara-manager/client/src/components/common/Dialog/Dialog.js:143]: Oh, my bad 🤦‍♂️. Thanks for fixing.

Comment (konekoya) [reply ohara-manager/client/src/components/common/Dialog/Dialog.js:143]: Not at all :)

**Pull request merged**